biogrid/apps/biogrid-mvp/src/assets/icons/battery-large.svg

```diff
@@ -0,0 +1 @@
+<svg id="Capa_1" enable-background="new 0 0 512 512" height="512" viewBox="0 0 512 512" width="512" xmlns="http://www.w3.org/2000/svg"><g><path d="m482 72h-51v-57c0-8.284-6.716-15-15-15h-64c-8.284 0-15 6.716-15 15v57h-162v-57c0-8.284-6.716-15-15-15h-64c-8.284 0-15 6.716-15 15v57h-51c-16.542 0-30 13.458-30 30v42c0 16.542 13.458 30 30 30h3v210c0 16.542 13.458 30 30 30h386c16.542 0 30-13.458 30-30v-210h3c16.542 0 30-13.458 30-30v-42c0-16.542-13.458-30-30-30zm-115-42h34v42h-34zm-256 0h34v42h-34zm-48 354v-34h386v34zm386-64h-386v-146h386zm33-176h-452l-.019-41.999s.006-.001.019-.001h452z"/><path d="m312 262h9v9c0 8.284 6.716 15 15 15s15-6.716 15-15v-9h9c8.284 0 15-6.716 15-15s-6.716-15-15-15h-9v-9c0-8.284-6.716-15-15-15s-15 6.716-15 15v9h-9c-8.284 0-15 6.716-15 15s6.716 15 15 15z"/><path d="m152 262h48c8.284 0 15-6.716 15-15s-6.716-15-15-15h-48c-8.284 0-15 6.716-15 15s6.716 15 15 15z"/></g></svg>
```

Comment:
![Screenshot_2020-07-27_072200](https://user-images.githubusercontent.com/18239203/88536635-2f345380-cfda-11ea-8c38-4ea1b906c888.jpg)

Reply:
Can we add a link to this screenshot in the file itself?